src/Integration/Binding/ProjectBindingOperation.cs

```diff
@@ -128,42 +112,39 @@ public void Commit()
         #endregion
 
         #region Helpers
+      
         private void CaptureProjectInformation()
         {
-            Debug.Assert(Integration.ProjectSystemHelper.IsCSharpProject(this.initializedProject) || Integration.ProjectSystemHelper.IsVBProject(this.initializedProject), "Unexpected project kind");
-            this.ProjectGroup = Integration.ProjectSystemHelper.IsCSharpProject(this.initializedProject) ? RuleSetGroup.CSharp : RuleSetGroup.VB;
+            this.ProjectGroup = GetProjectGroup(this.initializedProject);
             this.ProjectFullPath = this.initializedProject.FullName;
         }
 
         private void CalculateRuleSetInformation()
```

Comment:
The code was moved to ISolutionRuleSetsInformationProvider (and modified to be somewhat more general).